package-lock.json

```diff
@@ -263,54 +267,82 @@
 				}
 			}
 		},
+		"node_modules/@angular-devkit/core/node_modules/jsonc-parser": {
+			"version": "3.2.1",
+			"resolved": "https://registry.npmjs.org/jsonc-parser/-/jsonc-parser-3.2.1.tgz",
+			"integrity": "sha512-AilxAyFOAcK5wA1+LeaySVBrHsGQvUFCDWXKpZjzaL0PqW+xfBOttn8GNtWKFWqneyMZj41MWF9Kl6iPWLwgOA==",
+			"dev": true,
+			"license": "MIT"
+		},
+		"node_modules/@angular-devkit/core/node_modules/picomatch": {
+			"version": "4.0.1",
+			"resolved": "https://registry.npmjs.org/picomatch/-/picomatch-4.0.1.tgz",
+			"integrity": "sha512-xUXwsxNjwTQ8K3GnT4pCJm+xq3RUPQbmkYJTP5aFIfNIvbcc/4MUxgBaaRSZJ6yGJZiGSyYlM6MzwTsRk8SYCg==",
+			"dev": true,
+			"license": "MIT",
+			"engines": {
+				"node": ">=12"
+			},
+			"funding": {
+				"url": "https://github.com/sponsors/jonschlinkert"
+			}
+		},
 		"node_modules/@angular-devkit/core/node_modules/source-map": {
 			"version": "0.7.4",
+			"resolved": "https://registry.npmjs.org/source-map/-/source-map-0.7.4.tgz",
+			"integrity": "sha512-l3BikUxvPOcn5E74dZiq5BGsTb5yEwhaTSzccU6t4sDOH8NWJCstKO5QT2CvtFoK6F0saL7p9xHAqHOlCPJygA==",
 			"dev": true,
 			"license": "BSD-3-Clause",
 			"engines": {
 				"node": ">= 8"
 			}
 		},
 		"node_modules/@angular-devkit/schematics": {
-			"version": "16.2.0",
+			"version": "17.3.8",
+			"resolved": "https://registry.npmjs.org/@angular-devkit/schematics/-/schematics-17.3.8.tgz",
+			"integrity": "sha512-QRVEYpIfgkprNHc916JlPuNbLzOgrm9DZalHasnLUz4P6g7pR21olb8YCyM2OTJjombNhya9ZpckcADU5Qyvlg==",
 			"dev": true,
 			"license": "MIT",
 			"dependencies": {
-				"@angular-devkit/core": "16.2.0",
-				"jsonc-parser": "3.2.0",
-				"magic-string": "0.30.1",
+				"@angular-devkit/core": "17.3.8",
+				"jsonc-parser": "3.2.1",
+				"magic-string": "0.30.8",
 				"ora": "5.4.1",
 				"rxjs": "7.8.1"
 			},
 			"engines": {
-				"node": "^16.14.0 || >=18.10.0",
```

Comment:
here's the reason we had to update @nestjs/cli - because the depedency @angular-devkit/xxx needed to be updated